### PR TITLE
typos in README of pkg

### PR DESCRIPTION
**Pull request by luojie233**
**- What I did**

**- How I did it**

**- How to verify it**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by thaJeztah** [comment]
Please don't close, and reopen pull requests if you need to update them; you can update an existing PR without reopening a new one; for hints/tips read the contributors guide; https://docs.docker.com/opensource/workflow/work-issue/

**Comment by luojie233** [comment]
I reopen it.

**Comment by thaJeztah** [comment]
Can you squash your commits? Instructions can be found here https://docs.docker.com/opensource/workflow/work-issue/#/pull-and-rebase-frequently

**Comment by luojie233** [comment]
I squash one commit.

**Review by thaJeztah — APPROVED** [review]
LGTM, thanks!

File: pkg/plugins/pluginrpc-gen/README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ Where:
 - `--type` is the name of the interface to use
 - `--name` is the subsystem that the plugin "Implements"
 - `-i` is the input file containing the interface definition
-- `-o` is the output file where the the generated code should go
```
Comment (allencloud) [inline pkg/plugins/pluginrpc-gen/README.md]: Actually when I use a grep to match string `the the ` in the whole repo, I found that there are some more cases with this deuplicated `the`. Could you help us to fix those for us in this PR?

Comment (allencloud) [inline pkg/plugins/pluginrpc-gen/README.md]: Thanks for your work. Actually we usually do not change the code under directory of vendor, like file `vendor/github.com/davecgh/go-spew/spew/format.go` is in directory vendor. So, please revert those parts in your pr.

If we have to update those repo, we should make change in the upstream repo, for example repo go-spew in above sample. then use hack/vendor.sh to revendor that.

Comment (luojie233) [inline pkg/plugins/pluginrpc-gen/README.md]: I open a new PR #29623, this one could be closed.

**Comment by AkihiroSuda** [comment]
closing, according to https://github.com/docker/docker/pull/29585/files#r93370710

**Pull request closed without merging**